handlers.py

```diff
@@ -40,6 +47,37 @@ def handle_get_ssid(server, body, addr):
     server.send(json.dumps(resp).encode('utf-8'), addr)
 
 
+def register_car(userID):
+    """
+    Registers the car with the central server. Retries forever until the wanted
+    ACK is received. Returns the carID received in the ACK.
+
+    Arguments:
+    userID -- The ID of the user that is registering the car
+    """
+    server = Server(WLAN_IP, PORT, 60)
+    server.add_handler(MsgType.ACK, None)
+    attempts = 0
+    while True:
+        req = {"type": MsgType.REG_CAR, "user_id": userID}
+        server.send(json.dumps(req).encode('utf-8'), SERVER_ADDR)
+        # Wait for a max of 1 minute until the ACK from the server is received
+        start = time.time()
+        valid = False
+        while time.time() - start < 60:
```

Comment:
If the timeout is set to 60 already, then I don't think this loop is necessary.

Reply:
I made the loop in case the server receives some other unwanted packet before the receive times out, so that it'd go back to receiving until the 60 second mark is hit.

Reply:
Ah, I get it now. Now that I think about it, if the central server's response packet gets dropped, the car-pi probably shouldn't wait 60 seconds to retry though.

Reply:
Or if the car-pi's REG_USER packet gets dropped for example, it should only probably wait a couple of seconds before retrying.

Reply:
Okay, I'll just lower the retry time to like 5 seconds then. Yeah 60 seemed like a lot when I put it in originally but then I just forgot about it haha.